openpaas-james/mailbox/encrypted/api/src/main/scala/com/linagora/openpaas/encrypted/KeystoreManager.scala

```diff
@@ -10,7 +10,27 @@ object KeyId {
 
 case class KeyId(value: String)
 
-case class PublicKey(id: KeyId, payload: Array[Byte])
+class PublicKey(id: KeyId, payload: Array[Byte]) {
```

Comment:
Why not keeping the case class here? I don't see how is this commit useful^^'

Reply:
the different is the equals method

Reply:
> the different is the equals method

Bad answer. I prefer a separate "have same content method"

Reply:
For the array bytes? Scala can't compare two array bytes out of the box? (maybe it can't I don't know^^)